docs/creating_component_code_review.md

```diff
@@ -11,6 +11,7 @@ A checklist of things to do when you're adding a new component.
  1. Follow our [Style guidelines](development_guidelines.md)
  2. Use existing constants from [`const.py`](https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/const.py)
     * Only add new constants to `const.py` if they are widely used. Otherwise keep them on components level
+ 3. Rebase (**do not merge**) to [catch up with latest dev branch](development_catching_up.html)
```

Comment:
Let's drop this line to unblock this PR and we can discuss it in a future PR if we want.

Personally I don't think that it's better, as we are doing squash & merge, and so our repository won't be impacted. Rebase is going wrong too often. Yes, it's better for making sure no weird shit happens, but git error message suggests remote merging, and that's sadly causing a lot of havoc for people new to git. I think that putting it in here will raise the barrier unnecessarily.

Reply:
Sounds good.